tests/ext/test_updater.py

```diff
@@ -692,13 +701,13 @@ async def delete_webhook(*args, **kwargs):
     @pytest.mark.parametrize("ext_bot", [True, False])
     @pytest.mark.parametrize("drop_pending_updates", [True, False])
     @pytest.mark.parametrize("secret_token", ["SecretToken", None])
-    @pytest.mark.parametrize("unix", [None, True])
+    @pytest.mark.parametrize("unix", [None, "file_path", "socket_object"])
```

Comment:
my idea was:

```suggestion
    @pytest.mark.parametrize("unix", [None, "file_path", "socket_object"] if UNIX_AVAILABE else [None])
```

that would save the if clause below, I guess?

Reply:
I'm okay with leaving as is though …

Reply:
ohhhhh. That is so smart